en/docs/develop/restapis/applications.yaml

```diff
@@ -0,0 +1,2962 @@
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InboundProtocolsListResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/applications/{applicationId}/inbound-protocols/saml':
+    get:
+      tags:
+        - Inbound Protocols - SAML
+      summary: |
+        Retrieve SAML2 authentication protocol parameters of an application.
+      operationId: getInboundSAMLConfiguration
+      description: >
+        This API provides the capability to retrieve SAML2 authentication
+        protocol parameters of an application.
+      parameters:
+        - name: applicationId
+          in: path
+          description: Id of the application.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SAML2ServiceProvider'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+    put:
+      tags:
+        - Inbound Protocols - SAML
+      summary: |
+        Update SAML2 authentication protocol parameters of an application.
+      operationId: updateInboundSAMLConfiguration
+      description: >
+        This API provides the capability to store SAML2 authentication protocol
+        parameters of an application.
+
+        - There are three methods to create/update SAML2 authentication protocol configuration.
+            1. Metadata File (Need to send Base64 encoded content of the metadata file.)
```

Comment:
Metadata file (by sending the Base64 encoded content of the metadata file.) 

Remove the fullstops after "Metadata URL" and "Manual configuration"